package.json

```
@@ -33,7 +33,7 @@
     "http-proxy-agent": "^5.0.0",
     "https-proxy-agent": "^5.0.1",
     "is-potential-custom-element-name": "^1.0.1",
-    "nwsapi": "^2.2.2",
+    "nwsapi": "^2.2.3",
```

Version 2.2.3 introduced a regression involving `:not()` selectors, so I think it would be best to go straight to 2.2.4. See: https://github.com/dperini/nwsapi/issues/81

```suggestion
    "nwsapi": "^2.2.4",
```